scripts/prepare_db.py

```diff
@@ -33,10 +33,11 @@
         snakemake = mock_snakemake(
             "prepare_db",
             simpl="",
-            clusters="57",
-            ll="copt",
-            opts="Co2L5-72H",
+            clusters="287",
+            ll="c1.0",
+            opts="Co2L",
             planning_horizons="2030",
+            sopts="730H",
         )
 
     n0 = pypsa.Network(snakemake.input.network)

```

Comment:
Variable name "n0": why not just "n"?

Reply:
This script needs a lot of restructuring. Still not ready but I would keep it online because it helps a lot with testing